integration_test/cases/migrator.exs

```diff
@@ -81,6 +86,9 @@ defmodule Ecto.Integration.MigratorTest do
 
       assert count_entries() == 1
       assert [50] = run(TestRepo, path, :up, to: 50, log: false)
+
+      purge migrations
```

Comment:
Cool, why do we need those? Maybe we should just clean up the migration table before the tests? It would also solve the earlier [0] issue, wouldn't it?